test/e2e/testnet/testnet.go

```diff
@@ -332,16 +339,25 @@ func (t *Testnet) Start() error {
 			genesisNodes = append(genesisNodes, node)
 		}
 	}
+	// start genesis nodes asynchronously
 	for _, node := range genesisNodes {
-		err := node.Start()
+		err := node.StartAsync()
```

Comment:
In order to speed up the start-up phase, we can start all validator instances async and then wait for them being started.